code/datums/emergency_calls/colonist.dm

```diff
@@ -0,0 +1,43 @@
+
+
+//Blank colonist ERT for admin stuff.
+/datum/emergency_call/colonist
```

Comment:
Is this an intended change or merge garbage?

Reply:
this happened apparently because this file was deleted in a pr after i branched.